client/src/components/DisplayTable.js

```diff
@@ -74,8 +74,18 @@ export class DisplayTable extends React.Component {
           .every()
           .value()
       )
-      .sortBy( ({sort_values}) => sort_values[sort_by] )
-      .tap( descending ? _.noop : _.reverse )
+      .sortBy( ({sort_values}) => sort_values[sort_by] ? sort_values[sort_by] : Number.NEGATIVE_INFINITY )
```

Comment:
Oop, looks like this will sort values of 0 as - Inf

Reply:
Because 0's falsey. Probably want `_.isNumber(sort_values[sort_by])`.

Reply:
I guess blank < 0 then?